string[] name
float64[] position
int32 finger1
int32 finger2
int32 finger3
int32 finger4
int32 finger5


